brainspell/test_tornado.py

```diff
@@ -131,28 +134,51 @@ def test_procfile():
         assert filename in os.listdir()
 
 
-""" TODO: get selenium testing working
-def test_existence(): #Using selenium testing
+
+def test_existence(): #Using selenium testing to verify existence of site elements
     driver.get("https://brainspell.herokuapp.com")
-    assert "Brainspell" in driver.title #Checks website title is accurate
+    driver.implicitly_wait(3)
+    assert "Brainspell" in driver.title #Checks website was correctly received
 
     #Ensuring elements of view article page are all present
     driver.get("https://brainspell.herokuapp.com/view-article?id=00000000")
+    WebDriverWait(driver,3).until(
+        expected_conditions.presence_of_element_located((By.CLASS_NAME,"dropbtn"))
+    )
     meshButtons = driver.find_elements_by_class_name("dropbtn")
+
     assert meshButtons != None #Ensure Mesh terms are included
     table = driver.find_elements_by_class_name("experiment-table-row")
     assert table != None
 
-    #Click and vote on a MeSH tag
-    try:
-        myElem = WebDriverWait(driver,5)
-        buttons = driver.find_elements_by_class_name("dropbtn")
-        buttons[0].click()
-        assert "modal-body" in driver.page_source
-    except:
-        pass
+    #Examine Modal window that appears
+    buttons = driver.find_elements_by_class_name("dropbtn")
+    buttons[0].click()
+    WebDriverWait(driver,3).until(
+        expected_conditions.presence_of_element_located((By.CLASS_NAME,"modal-body"))
+    )
+    assert "modal-body" in driver.page_source
     driver.find_element_by_id("closer").click()
 
+
+    #Evaluate search page
+    driver.get("https://brainspell.herokuapp.com/search?q=brain&req=t")
+    show_widgets = WebDriverWait(driver,10).until(
+        expected_conditions.element_to_be_clickable((By.ID,"widgetOption"))
+    )
+    show_widgets.click()
+    assert show_widgets.value_of_css_property("display") == "none"
+
+    #Evaluate whether 10 results are properly returned
+    link = driver.find_elements_by_css_selector('[href^=view-article]')
+    assert len(link) == 10, "10 Search Results were not Found!"
```

Comment:
Why the strange capitalization here? Try to keep things consistent and grammatically correct